### ISSUE TITLE
Bisect Tag

### ISSUE THREAD

**Issue by ryneeverett** [ryneeverett/git-regress#5]

**Comment by ryneeverett** [comment]
This will require more thorough testing with more commits on the example repo. Postpone until after #9.

**Comment by ryneeverett** [comment]
This will require more thorough testing with more commits on the example repo. Postpone until after #15.

**Comment by ryneeverett** [comment]
This will require more thorough testing with more tags on the example repo. Postpone until after #14.

**Comment by ryneeverett** [comment]
Now that we have repo caching, #14 can no longer be considered a blocker.

**Comment by ryneeverett** [comment]
However, I don't see an easy way to implement this and I can't imagine being in a situation with so many tags that a linear search is too slow.
